mlflow/entities/trace_info.py

```diff
@@ -127,12 +127,23 @@ def from_proto(cls, proto) -> "TraceInfo":
 
             return TraceInfoV2.from_proto(proto).to_v3()
 
+        from mlflow.tracing.utils import construct_trace_id_v4
```

Comment:
Nit - can this be imported at the top of the module?

Reply:
That will introduce all the dependencies inside mlflow/tracing/utils/__init__.py while entities shouldn't depend on them, so we should put it inside here

Reply:
Can we add an inline comment explaining that?